sequencer/src/main/java/net/consensys/linea/sequencer/txselection/selectors/ProfitableTransactionSelector.java

```diff
@@ -202,4 +273,48 @@ private void rememberUnprofitable(final Transaction transaction) {
     unprofitableCache.add(transaction.getHash());
     log.atTrace().setMessage("unprofitableCache={}").addArgument(unprofitableCache::size).log();
   }
+
+  private void updateMetric(
+      final Phase label,
+      final TransactionEvaluationContext<? extends PendingTransaction> evaluationContext,
+      final Transaction tx,
+      final Wei profitablePriorityFeePerGas) {
+
+    maybeProfitabilityMetrics.ifPresent(
+        histogramMetrics -> {
+          final var effectivePriorityFee =
+              evaluationContext.getTransactionGasPrice().subtract(baseFee);
+          final var ratio =
+              effectivePriorityFee.getValue().doubleValue()
+                  / profitablePriorityFeePerGas.getValue().doubleValue();
```

Comment:
are we guaranteed that profitablePriorityFeePerGas is never zero.

Reply:
it could be zero only the pricing conf for variable and fixed cost is zero, that seems unreasonable, and according to the Java rules that operation will result in Double.POSITIVE_INFINITY